docker-compose.yaml

```diff
@@ -0,0 +1,92 @@
+version: "3"
+
+networks:
+  driftnet:
+
+
+volumes:
+  app_data: {}
+
+services:
+  prometheus:
+    image: prom/prometheus:v2.34.0
+    volumes:
+      - ./infra/prometheus/:/etc/prometheus/
+    command:
+      - "--config.file=/etc/prometheus/prometheus.yml"
+      - "--storage.tsdb.path=/prometheus"
+    ports:
+      - 9090:9090
+    networks:
+      - driftnet
+  grafana:
+    image: grafana/grafana:8.4.5
```

Comment:
sir, do you not enjoy grafana 9+?

Reply:
Good point - switched to using latest for all images